CONTRIBUTING.md

```diff
@@ -21,14 +21,14 @@ start participating.
 
 ## Reporting security issues
 
-The Docker maintainers take security seriously. If you discover a security
+The Moby maintainers take security seriously. If you discover a security
 issue, please bring it to their attention right away!
 
 Please **DO NOT** file a public issue, instead send your report privately to
 [security@docker.com](mailto:security@docker.com).
```

Comment:
Should there be a non-Docker alias for this?

Reply:
there will be shortly, it hadnt been set up yesterday, will check again today...